dependency-check-suppressions.xml

```diff
@@ -1,11 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?><suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
   <suppress>
     <notes>Temporary Suppression
-        CVE-2021-37533 refer [Ticket]
-        CVE-2021-4277 refer [Ticket]
-        CVE-2022-3064 refer https://tools.hmcts.net/jira/browse/CCD-4157
-        CVE-2021-4235 refer https://tools.hmcts.net/jira/browse/CCD-4157
-        CVE-2022-45143 refer [Ticket]</notes>
+        CVE-2022-45143 refer https://tools.hmcts.net/jira/browse/CCD-4312</notes>
```

Comment:
The ticket is for a change to CPO but with the same cve. Maybe the ticket can be updated to add this service

Reply:
Thanks Marvin.  I've added Test Stub Service (and also Manage Case Assignment) to CCD-4312 Components.